src/scripts/main.js

```diff
@@ -1,3 +1,67 @@
 'use strict';
 
-// write code here
+const url = 'https://mate-academy.github.io/'
+  + 'phone-catalogue-static/api/phones';
```

Comment:
```suggestion
const BASE_URL = 'https://mate-academy.github.io/phone-catalogue-static/api';
```
для чого ти додаєш строки? 
зручніше буде не прописувати тут `phones` а передавати рядок в функцію fetchData параметром, тоді ця функція може бути використана для запиту з а чим завгодно з цього api
і я б назвав це BASE_URL, тому що це базовий url який потім можна буде модифікувати для різних запитів
ну і написав б використовуючи snake case і великими літерами тому що це константа

Reply:
Чому додавав? Я спочатку бавився з варіантом адреси і назви змінної і довжина строки була завелика, тому додав, потім як вкоротив то не звернув уваги пройде без додавання